docs/astro.config.mjs

```diff
@@ -300,6 +300,11 @@ export default defineConfig({
               translations: { es: 'Dividir un pago entrante' },
               link: '/guides/split-payments/'
             },
+            {
+              label: 'One-time payment with debitAmount',
+              translations: { es: 'Pago único con debitAmount' },
```

Comment:
```suggestion

```

Reply:
I'd rather leave all translations up to the translation service so they don't see this and think they don't need to review it.